timely_beliefs/beliefs/classes.py

```diff
@@ -162,11 +174,34 @@ def sensor_id(cls):
     def source_id(cls):
         return Column(Integer, ForeignKey("belief_source.id"), primary_key=True)
 
-    def __init__(self, sensor, source, value: float, **kwargs):  # TODO: type hinting?
+    def __init__(
+        self,
+        sensor: DBSensor,
+        source: DBBeliefSource,
+        value: float,
+        cumulative_probability: Optional[float] = None,
+        cp: Optional[float] = None,
+        sigma: Optional[float] = None,
+        event_start: Optional[datetime] = None,
+        event_time: Optional[datetime] = None,
+        belief_horizon: Optional[timedelta] = None,
+        belief_time: Optional[datetime] = None,
+    ):
         self.sensor_id = sensor.id
         self.source_id = source.id
-        TimedBelief.__init__(self, sensor=sensor, source=source, value=value, **kwargs)
-        Base.__init__(self)
```

Comment:
This is actually the small fix: a tb Base is initialized where it shouldn't be. It's a bit hidden within this change list, so that's why this comment is here.

Reply:
I believe you're correct. This want caught because Base was imported to the file, but I'm not sure what calling Base.__init__ really did in the Mixin. Thanks!